src/components/illustration/illustration.stories.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { Illustration } from './illustration';
+import { ILLUSTRATION_NAMES } from './illustration-set';
+import { IllustrationPreview, PreviewGrid } from './illustration-preview';
+
+export default {
+  component: Illustration,
+  title: 'Components/Illustration',
+  parameters: {
+    componentSubtitle: 'Helps to show one of our illustrations',
+  },
+  /**
```

Comment:
I had argTypes defined here, but they really don't coexist well with the automatically extracted values. Plus they seem to bleed into any story, even if excluded. So weird. I removed it here  as the automatic extraction works okay for the component itself and I define only a single field for the overview story later.

Reply:
If you want these `argTypes` only on the `Regular` story you could add them there instead of here in this "global" config.

Reply:
I tried this and yes it works. I was curious then. When will the docs engine read it? And what I found is weird.
That parameter is ignored when it's placed on the overview story. I though it's related to the export order but changing this would not work either. It's only accepted on the "normal" story and on the default export. Maybe the `as any` of the overview story makes things worse but at some point I stopped testing this. I will check in future stories again how this behaves.

The more correct place is the default export I think, because that's describing the overall component we are dealing with, so I keep it.